controllers/animal_c.js

```diff
@@ -1,10 +1,56 @@
 const db = require('../database/data');
+const ObjectId = require('mongodb').ObjectId;
 const animalController = {};
 
 // Animal Endpoints:
 // - GET /animals
+
+animalController.getAllAnimals = async function (req, res) {
+    try{
+        const result = await db.getDatabase.db('WildlifeAPI').collection('Animals').find();
+        result.toArray().then((animals) => {
+            res.setHeader('Content-Type', 'application/json');
+            res.status(200).json(animals);
+        })
+    } catch {
+        console.log(`animals/getAll error: ${error}`)
+        res.status(500).send('There was an error retrieving animals.');
+    }
```

Comment:
I think we need to catch the `error` so we can `console.log` it. In any event, I'd prefer to remove `console.log`s from our code and use the debugger instead. Another alternative would be to use `console.log`s only in our local environments. Consider adding the no-console ESLint rule to warn us just in case. Last, for consistency, perhaps we should consider returning JSON in our `catch` blocks as well.

```suggestion
    } catch (error) {
        console.log(`animals/getAll error: ${error}`)
        res.status(500).json({ message: 'There was an error retrieving animals.' });
    }
```

Reply:
We already send strings in other places where we are not returning database results. I see no reason to break that convention.